assets/app/view/create_game.rb

```diff
@@ -110,7 +114,7 @@ def submit
         end
 
         create_hotseat(
-          players: players,
+          players: players.map { |name| { name: name } },
```

Comment:
why is this changed? was it always broken?

Reply:
Nope, but when I was doing the whitespace stripping below, I ended up with code like `.map { |name| name[:name].gsub(/\s+/, ' ').strip }` which was just too many "names" in a row.

I thought, "why not map to parameter format down inside `create_hotseat` where it's required, and let the rest of the code deal with the player names as an array of strings." Perhaps I should have renamed the array of strings to `player_names`?

Reply:
ah i see it now thanks, this is fine